src/keyboard.js

```diff
@@ -82,14 +83,34 @@ export function getCode( key ) {
  */
 export function parseKeystroke( keystroke ) {
 	if ( typeof keystroke == 'string' ) {
-		keystroke = keystroke.split( /\s*\+\s*/ );
+		keystroke = splitKeystrokeText( keystroke );
 	}
 
 	return keystroke
 		.map( key => ( typeof key == 'string' ) ? getCode( key ) : key )
 		.reduce( ( key, sum ) => sum + key, 0 );
 }
 
+/**
+ * It translates any keystroke string text like `"CTRL+A"` to an
+ * environment–specific keystroke, i.e. `"⌘A"` on Mac OSX.
+ *
+ * @method utils.keyboard.getEnvKeystrokeText
+ * @param {String} keystroke Keystroke text.
+ * @returns {String} Keystroke text specific for the environment.
+ */
+export function getEnvKeystrokeText( keystroke ) {
+	const split = splitKeystrokeText( keystroke );
+
+	if ( env.mac ) {
+		if ( split[ 0 ].toLowerCase() == 'ctrl' ) {
```

Comment:
I'd document that because I've just found this in our code:

```

		// Set the Ctrl+B keystroke.
		editor.keystrokes.set( 'CTRL+B', 'bold' );
```

And compare this with `parseKeystroke` documentation... :D